### ISSUE TITLE
Delete old stale files

### ISSUE THREAD

**Issue by rajeeja** [ECP-CANDLE/Supervisor#26]
mlrMBO1 and mlrMBO3 
ai_workflow
workflowx.sh.. etc

**Comment by jmjwozniak** [comment]
Yes, I suggest we do this after the merge to master, which will be after the cancer workshop deadline.  In the meantime, feel free to mark them as obsolete or even put poison pills in them.
